tiledb/sm/buffer/buffer.cc

```diff
@@ -27,13 +27,12 @@
  *
  * @section DESCRIPTION
  *
- * This file implements class Buffer.
+ * This file implements classes BufferBase and Buffer.
```

Comment:
Also `ConstBuffer` and `PreallocatedBuffer`.

Reply:
Done